### PR TITLE
chore: add missing data

### PR DESCRIPTION
**Pull request by acbeni**
Tip for review, skip the generated code and go straight for the tests 😄

### CODE REVIEW

File: packages/ml-sdk/src/generated/models/missing-data.ts [hunk 1]
```diff
@@ -84,8 +74,8 @@ export interface MissingAttributes {
   readonly attributeCoverage?: AttributeCoverage
 }
 export interface MissingAttributesMeta {
-  readonly productLevel: MissingAttributesProductLevel
-  readonly variantLevel: MissingAttributesVariantLevel
+  readonly productLevel: MissingAttributesDetails
+  readonly variantLevel: MissingAttributesDetails
```
Comment (evilaz) [inline packages/ml-sdk/src/generated/models/missing-data.ts]: why is this renamed? is it just internal? 🤔

Comment (acbeni) [reply packages/ml-sdk/src/generated/models/missing-data.ts]: actually here the types are the same, they just have different descriptions, this is why i decided to make the same type for both.

**Comment by acbeni** [comment]
Do you have any other questions here @evilaz ?

**Comment by mbarga** [comment]
In addition to attributes we also have missing images and prices. Wonder if its also worth making tests for the other two endpoints?

**Comment by acbeni** [comment]
I will add them in a separate PR @mbarga .

**Comment by evilaz** [comment]
> Do you have any other questions here @evilaz ?

No I'm good, I don't get much anyway was just curious :)
